app/src/main/java/com/example/dragonx/view/fragments/RocketDetailsFragment.kt

```diff
@@ -0,0 +1,50 @@
+package com.example.dragonx.view.fragments
+
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModelProvider
+import com.denzcoskun.imageslider.constants.ScaleTypes
+import com.denzcoskun.imageslider.models.SlideModel
+import com.example.dragonx.R
+import com.example.dragonx.models.Rocket
+import com.example.dragonx.viewmodel.RocketDetailsViewModel
+import com.example.dragonx.viewmodel.RocketListViewModel
+import kotlinx.android.synthetic.main.fragment_rocket_details.view.*
+import kotlinx.coroutines.*
+
+class RocketDetailsFragment : Fragment() {
+    private lateinit var viewModel: RocketDetailsViewModel
```

Comment:
why you don't initial viewmodel here?

Reply:
I just stuck the tutorial video on Udacity )

Reply:
In this situation you reinitial viewmodel when view recreate( f.e. when app rotate). Are you sure you need there?